backend/src/test/java/com/woowacourse/levellog/presentation/FeedbackControllerTest.java

```diff
@@ -46,7 +41,8 @@ void save_alreadyExist_exceptionThrown() throws Exception {
                     .willThrow(new FeedbackAlreadyExistException("피드백이 이미 존재합니다."));
 
             // when
-            final ResultActions perform = requestPost("/api/levellogs/" + levellogId + "/feedbacks", token, request);
+            final ResultActions perform = requestPost("/api/levellogs/" + levellogId + "/feedbacks",
+                    request);
```

Comment:
```suggestion
            final ResultActions perform = requestPost("/api/levellogs/" + levellogId + "/feedbacks", request);
```
다른 데서도 request 전에 개행을 하고 있네요👀